README.md

```diff
@@ -413,45 +475,114 @@ This field SHOULD NOT be used to sign arbitrary data, such as signature challeng
+
+```json
+"https://example.com",
+"https://example.com/blog"
+```
+
+#### 3.2.6.2 Abilities
+
+Abilities MUST be presented as a string. By convention, abilities SHOULD be namespaced with a slash, such as `msg/send`. One or more abilities MUST be given for each resource.
+
+#### 3.2.6.3 Caveat Array
+
+Caveats MAY be open ended. Caveats MUST be understood by the executor of the eventual [invocation]. Caveats MUST prevent invocation otherwise. Caveats MUST be formatted as objects.
+
+On validation, the caveat array MUST be treated as a logically disjunct (an "OR", NOT an "and"). In other words: passing validation against _any_ caveat in the array MUST pass the check. For example, consider the following capabilities:
+
+```json
+{
+  "dns:example.com?TYPE=TXT": {
+    "crud/create": [{}]
+  },
+  "https://example.com/blog": {
+    "crud/read": [{}],
+    "crud/update": [
+      {"status": "draft"},
+      {"status": "published", "day-of-week": "Monday"} // only publish on Mondays
+    ]
+  }
+}
+```
+
+The above MUST be interpreted as the set of capabilities below. If _any_ are matched, the check MUST pass validation.
+
+| Resource                   | Ability       | Caveat                                            |
+|----------------------------|---------------|---------------------------------------------------|
+| `dns:example.com?TYPE=TXT` | `crud/create` | Always                                            |
+| `https://example.com/blog` | `crud/read`   | Always                                            |
+| `https://example.com/blog` | `crud/update` | `{status: "draft"}`                               |
+| `https://example.com/blog` | `crud/update` | `{status: "published", "day-of-week": "monday"}` |
+
+The caveat array SHOULD NOT be empty, as an empty array means "in no case" (which is equivalent to not listing the ability). This follows from the rule that delegations MUST be of equal or lesser scope. When an array is given, an attenuated caveat MUST (syntactically) include all of the fields of the relevant proof caveat, plus the newly introduced caveats.
+
+| Proof Caveats | Delegated Caveats | Is Valid? | Comment                              |
+|---------------|-------------------|-----------|--------------------------------------|
+| `[{}]`        | `[{}]`            | Yes       | Equal                                |
+| `[x]`         | `[x]`             | Yes       | Equal                                |
+| `[x]`         | `[{}]`            | No        | Escalation to any                    |
+| `[{}]`        | `[x]`             | Yes       | Introduces a new caveat              |
+| `[x, y]`      | `[x]`             | Yes       | Removes a capability                 |
+| `[x, y]`      | `[x, (y + z)]`    | Yes       | Attenuates existing caveat           |
+| `[x, y]`      | `[x, y, z]`       | No        | Escalation by adding new capability  |
```

Comment:
I think a line in here about 
`[x] -> [y]` which would not be allow unless `y` is an attenuation of `x`

I also might slightly change the wording around
`[{}] -> [x]` because you are not "introducing a new caveat", you are "attenuating `[{}]` to `[x]`"